pkg/cli/topo.go

```diff
@@ -90,46 +59,46 @@ func runAddEntityCommand(cmd *cobra.Command, args []string) error {
 
 func getGetRelationCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "relation <id>",
-		Args:  cobra.MinimumNArgs(1),
-		Short: "Get a topo relationship",
-		RunE:  runGetRelationCommand,
+		Use:     "relation <id>",
+		Aliases: []string{"relations"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Get topo relationships",
+		RunE:    runGetRelationCommand,
 	}
 
 	return cmd
 }
 
 func runGetRelationCommand(cmd *cobra.Command, args []string) error {
+	return runGetCommand(cmd, args, topo.Object_RELATIONSHIP)
+}
 
+func runGetCommand(cmd *cobra.Command, args []string, objectType topo.Object_Type) error {
 	var objects []*topo.Object
 
-	outputWriter := cli.GetOutput()
-	writer := new(tabwriter.Writer)
-	writer.Init(outputWriter, 0, 0, 3, ' ', tabwriter.FilterHTML)
+	updates := make(chan *topo.Update)
+	done := make(chan bool)
+	defer close(updates)
+	defer close(done)
+
+	go printIt(updates, objectType, done)
 
-	objects, err := readObjects(cmd, args)
+	objects, err := readObjects(cmd, args, objectType)
 	if err != nil {
 		return err
 	}
 
-	if len(objects) != 0 {
-		switch obj := objects[0].Obj.(type) {
-		case *topo.Object_Relationship:
-			_, _ = fmt.Fprintf(writer, "ID\t%s\n", objects[0].Ref.GetID())
-			_, _ = fmt.Fprintf(writer, "type\t%s\n", obj.Relationship.GetType())
-			for _, ref := range obj.Relationship.GetSourceRefs() {
-				_, _ = fmt.Fprintf(writer, "src-entity-id\t%s\n", string(ref.ID))
-			}
-			for _, ref := range obj.Relationship.GetTargetRefs() {
-				_, _ = fmt.Fprintf(writer, "tgt-entity-id\t%s\n", string(ref.ID))
-			}
-		case nil:
-			cli.Output("Error: nil object\n")
-		default:
-			cli.Output("Error: get error\n")
+	for _, obj := range objects {
+		updates <- &topo.Update{
+			Type:   topo.Update_UNSPECIFIED,
+			Object: obj,
 		}
 	}
-	return writer.Flush()
+
+	updates <- &topo.Update{}
+	<-done
+
+	return nil
 }
 
 func getAddRelationCommand() *cobra.Command {
```

Comment:
I think the `type` flag on line 111 is not used - if it is, it should say the `type of relationship`